packages/server/src/sdk/app/rows/search/sqs.ts

```diff
@@ -172,7 +172,8 @@ export async function search(
       sql = sql.replace(/`doc2`.`rowId`/g, "`doc2.rowId`")
 
       const db = context.getAppDB()
-      return await db.sql<Row>(sql, bindings)
+      const rows = await db.sql<Row>(sql, bindings)
```

Comment:
❤️  finally somebody adding this instead of removing mines haha

Reply:
It made placing a breakpoint easier 😅